modules/031-linstor/monitoring/prometheus-rules/drbd-devices.yaml

```diff
@@ -0,0 +1,158 @@
+- name: kubernetes.drbd.device_state
+  rules:
+    - alert: LinstorVolumeIsNotHealthy
```

Comment:
Alerts should start with D8. We rely on it to send alerts to the Deckhouse supporting team (not to end platform users).